ui_testing/pages/testplan_page.py

```diff
@@ -240,3 +240,13 @@ def get_testunit_quantification_limit(self, testunits, testunit_display_name):
                 quantification_limit = testunit['Quantification Limit']
                 return quantification_limit
 
+    def edit_test_unit_quantification_limits(self, upper, lower):
+        self.navigate_to_testunits_selection_page()
+        elems = self.base_selenium.find_elements('general:col_6')
+        old_upper = self.base_selenium.find_element_in_element(source=elems[4], destination_element='general:input')
+        old_lower = self.base_selenium.find_element_in_element(source=elems[5], destination_element='general:input')
```

Comment:
Move elements values to the elements.py file

Reply:
elements already in elements.py file

Reply:
my bad